pykoi/frontend/src/lib/Chatbots/Components/DownloadModal.svelte

```diff
@@ -34,6 +34,19 @@
         }
     };
 
+    const handleSubmit = async (e) => {
```

Comment:
Nice!. May be worth destucturing the `file_exists` directly, since we're just using file_exists there
 e.g.   
  `const { file_exists } = await response.json();`
  
  then
    
```
    if (file_exists) {
        ...
    } else {
        ...
    }
};
```

Reply:
Just updated with this!